src/SwiftReachApi/SwiftReachApi.php

```diff
@@ -32,26 +32,55 @@ class SwiftReachApi
      */
     private $base_url;
 
+    /**
+     * @var array
+     * default request options
+     */
+    private $defaults;
 
     /**
      * @var \GuzzleHttp\Client
      */
     private $guzzle_client;
 
-    public function __construct($api_key, $base_url = "http://api.v4.swiftreach.com")
+    /**
+     * SwiftReachApi constructor.
+     * @param $api_key
+     * @param string $base_url
+     * @param array $defaults
+     */
+    public function __construct($api_key, $base_url = "http://api.v4.swiftreach.com", array $defaults = [])
     {
+        $default_defaults = $this->getDefaultDefaults();
+
+        $defaults = array_merge(
+            $default_defaults,
+            $defaults
+        );
+
         $this->setApiKey($api_key);
         $this->setBaseUrl($base_url);
+        $this->setDefaults($defaults);
 
         $this->guzzle_client = new Client(
             array(
-                "defaults" => array(
-                    "timeout"         => 30,
-                    "connect_timeout" => 30
-                )
+                "defaults" => $default_defaults
             )
         );
     }
+
+    /**
+     * @return array
+     */
+    protected function getDefaultDefaults()
+    {
+        $default_defaults = [
+            "timeout"         => 30,
```

Comment:
can you lower this to 5 seconds

Reply:
sure. these are the default options, so i figured i'd leave them alone, and set it to 5 seconds on the ATD side. but those can just use the defaults of 5 if that's the case; no ATD changes needed.